packages/hull/src/handlers/external-handler.js

```
@@ -78,7 +78,7 @@ const handlerFactory = ({
     }
 
     // There'a an actual Response to send
-    const { pageLocation, data, status = 200, text } = response;
+    const { pageLocation, data, error, status = 200, text } = response;
```

where are we using this ability to return an error in the response?  Not sure I like delegating more responsibility to the underlying code to catch errors, but would need to see the case.  Might be an ok feature, but means there's more catching in underlying code which sometimes can lead to suppressing errors

---

Checkout NEW_README.md, usage is explained. It's a way to easily return an error with a predictable pattern: `status: 500, { error: xxx }`

If we ever decide to make them different, we won't have to change the return data in each connector We can just update the handler middlewares